src/SymbolDirectories/ManagerTest.cpp

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <QCoreApplication>
+#include <filesystem>
+
+#include "SymbolDirectories/Manager.h"
+
+const std::filesystem::path directory0{"/path/to/symbols/directory"};
+const std::filesystem::path directory1{"/home/src/project/build/"};
+const std::filesystem::path directory2{R"(c:\project\build\)"};
+
+constexpr const char* kOrgName = "The Orbit Authors";
+
+namespace orbit_symbol_directories {
+
+TEST(SymbolDirectoriesManager, SetAndGet) {
+  QCoreApplication::setOrganizationDomain(kOrgName);
+  QCoreApplication::setApplicationName("SymbolDirectoryManager.SetAndGet");
+
+  Manager manager{};
+
+  std::vector<std::filesystem::path> directories{
+      directory0,
+      directory1,
+      directory2,
+  };
+
+  manager.SetDirectories(directories);
+  EXPECT_EQ(manager.GetDirectories(), directories);
```

Comment:
You could also clear the directories afterwards and see if that worked.

Reply:
I assume you mean `.SetDirectories({})` ? I added that